src/inputs/internal/SelectFieldBase.tsx

```diff
@@ -61,19 +63,36 @@ export function SelectFieldBase<O, V extends Value>(props: BeamSelectFieldBasePr
     disabled,
     readOnly,
     onSelect,
-    options: maybeOptions,
+    options,
     multiselect = false,
-    getOptionLabel,
-    getOptionValue,
-    getOptionMenuLabel = getOptionLabel,
+    getOptionLabel: gol,
+    getOptionValue: gov,
+    getOptionMenuLabel: goml,
```

Comment:
Hm, maybe just don't destructure these, and where we do `gol`, just do `props.getOptionLabel`?